scripts/deploy/db-sync

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+IFS=$'\n\t'
+set -euo pipefail
+
+#
+# GovCMS database sync
+#
+# This will pull a database from the production
+# environment.
+#
+
+LAGOON_ENVIRONMENT_TYPE=${LAGOON_ENVIRONMENT_TYPE:-production}
+GOVCMS_DEPLOY_WORKFLOW_CONTENT=${GOVCMS_DEPLOY_WORKFLOW_CONTENT:-retain}
+
+if [ "$LAGOON_ENVIRONMENT_TYPE" = "production" ]; then
+  exit 0
+fi
+
+if [ "$LAGOON_ENVIRONMENT_TYPE" = "local" ]; then
+  exit 0
+fi
+
+BOOTSTRAP=$(drush status --fields=bootstrap | grep -c "Successful")
+
+# Synchronise the environment on the first deploy.
+if [ "$BOOTSTRAP" -eq 1 ] && [ "$GOVCMS_DEPLOY_WORKFLOW_CONTENT" != "import" ]; then
```

Comment:
Is this overrideable - e.g. for canary/test environments, there is merit in making them sync from prod regularly to "stay fresh"?

Reply:
Yeah definitely agree - if we set `GOVCMS_DEPLOY_WORKFLOW_CONTENT` on the canary environment to `import` it should always pull back a fresh db.